tests/datasource/fluent/test_batch.py

```diff
@@ -290,3 +294,57 @@ def test_batch_validate_expectation_suite_does_not_persist_a_batch_definition(
 
     assert result.success
     assert len(asset.batch_definitions) == 0
+
+
+@pytest.mark.filesystem
+def test_batch_compute_metrics_single_metric_success(
+    pandas_setup: Tuple[AbstractDataContext, Batch],
+):
+    _, batch = pandas_setup
+    metric = ColumnValuesBetween(
+        batch_id=batch.id,
```

Comment:
I'm wondering if we need to pass in `batch.id` since we could infer that from the `batch` when we call `batch.compute_metrics(...)`.

Reply:
Yes, I have been planning to take `batch_id` private, but I am going to do it in it's own PR.